src/sentry/models/groupassignee.py

```diff
@@ -204,7 +204,7 @@ def assign(
     def deassign(
         self,
         group: Group,
-        acting_user: User | RpcUser | None = None,
+        acting_user: User | RpcUser | None,
```

Comment:
Down below we call `create_group_activity` which expects `RpcUser`
https://github.com/getsentry/sentry/blob/0d337eb5e24a7c70a2d8ea9c896b4228ee71e5ce/src/sentry/models/activity.py#L81-L85